Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6-alpine
+ENV PYTHONUNBUFFERED 1  
+
+RUN apk update && \
+    apk add --virtual build-deps gcc python-dev musl-dev && \
+    apk add postgresql-dev bash
+
+RUN mkdir /config  
+ADD /config/requirements.txt /config/  
+RUN pip install -r /config/requirements.txt
+RUN mkdir /src
+WORKDIR /src
+EXPOSE 80
+CMD ["/bin/ash", "./wait_for_postgres.sh", "gunicorn interactivemap.wsgi -b 0.0.0.0:80"]
```

Comment:
Curious: why `/bin/ash`? Also, the script already has a shebang /bin/bash, so the first part might not be necessary?

Reply:
`/bin/bash` doesn't exist in Alpine linux, so we have to use either `/bin/ash` or `/bin/sh`.  It's best to avoid marking the script as executable and instead just specifying the interpreter so I think I'll just get rid of the shebang line. The `/bin/bash` as opposed to `/bin/ash` was a force of habit, woops.

Reply:
Makes sense.